src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaverImplementation.android.cs

```diff
@@ -19,10 +22,19 @@ public async Task<string> SaveAsync(string initialPath, string fileName, Stream
 			throw new PermissionException("Storage permission is not granted.");
 		}
 
+		const string baseUrl = "content://com.android.externalstorage.documents/document/primary%3A";
+		if (Android.OS.Environment.ExternalStorageDirectory is not null)
+		{
+			initialPath = initialPath.Replace(Android.OS.Environment.ExternalStorageDirectory.ToString(), string.Empty, StringComparison.InvariantCulture);
```

Comment:
Let's use the `AbsolutePath` prop. instead
```suggestion
			initialPath = initialPath.Replace(Android.OS.Environment.ExternalStorageDirectory.AbsolutePath, string.Empty, StringComparison.InvariantCulture);
```